unittests/multisig_tests.cpp

```diff
@@ -151,19 +151,19 @@ class eosio_msig_tester : public tester {
          action act;
          act.account = N(eosio.msig);
          act.name = name;
-         act.data = abi_ser.variant_to_binary( action_type_name, data );
+         act.data = abi_ser.variant_to_binary( action_type_name, data, control->get_abi_serializer_max_time_ms() );
```

Comment:
get_abi_serializer_max_time_ms of chain_controller is already removed (this piece of code is actually just inside a comment)